canlink/handler.go

```diff
@@ -0,0 +1,6 @@
+package canlink
+
+type Handler interface {
```

Comment:
I don't fully understand how this will be used. Could you document this a bit more and just maybe explain the reasoning for this design. The `busmanager` piece totally makes sense to me, I just can't picture how the handlers will be used.

Reply:
Handler doesn't necessarily need to exist. It's only use right now is to provide as an argument to `Register`. The idea is that objects like `Tracer` and `CanClient` would be handlers.

Reply:
I thought we were using Handlers as our interface for "handling" incoming messages, which would make Handler a key part of this architecture.